### ISSUE TITLE
Line shame emits invalid SVG

### ISSUE THREAD

**Issue by bbalet** [literallycanvas/literallycanvas#423]
I'll try my best to explain this issue although I don't know the tech and I hope this is not a duplicate.

I've tried to play with the demo and the export to SVG feature (http://literallycanvas.com/examples/export.html#exporting-to-svg)with this simple drawing :

![map](https://cloud.githubusercontent.com/assets/1221546/16987972/00423ec8-4e8f-11e6-8b6b-70a850c6daa2.png)

When I click on export, I get an error (XML document is not well formed). Below is the result of the export

```
data:image/svg+xml;base64,PHN2ZyB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnIHdpZHRoPSczMDcnIGhlaWdodD0nMzI1JyB2aWV3Qm94PScwIDAgMzA3IDMyNSc+IDxyZWN0IHdpZHRoPSczMDcnIGhlaWdodD0nMzI1JyB4PScwJyB5PScwJyBmaWxsPSd0cmFuc3BhcmVudCcgLz4gPGcgdHJhbnNmb3JtPSd0cmFuc2xhdGUoLTMyLCAtMjkpJz4gPGc+IDxsaW5lIHgxPSc2MicgeTE9JzMzLjUnIHgyPSc2MicgeTI9JzI3Ny41JyAgc3Ryb2tlLWxpbmVjYXA9J3JvdW5kJyBzdHJva2U9J2hzbGEoMCwgMCUsIDAlLCAxKSAnc3Ryb2tlLXdpZHRoPSc1JyAvPiAgPC9nPjxnPiA8bGluZSB4MT0nNjInIHkxPScyNzYuNScgeDI9JzM1JyB5Mj0nMzE2LjUnICBzdHJva2UtbGluZWNhcD0ncm91bmQnIHN0cm9rZT0naHNsYSgwLCAwJSwgMCUsIDEpICdzdHJva2Utd2lkdGg9JzUnIC8+ICA8L2c+PGc+IDxsaW5lIHgxPSc2NScgeTE9JzMyOS41JyB4Mj0nNjUnIHkyPSczNTQuNScgIHN0cm9rZS1saW5lY2FwPSdyb3VuZCcgc3Ryb2tlPSdoc2xhKDAsIDAlLCAwJSwgMSkgJ3N0cm9rZS13aWR0aD0nNScgLz4gIDwvZz48Zz4gPGxpbmUgeDE9JzY1JyB5MT0nMzU0LjUnIHgyPSczMzgnIHkyPSczNTMuNScgIHN0cm9rZS1saW5lY2FwPSdyb3VuZCcgc3Ryb2tlPSdoc2xhKDAsIDAlLCAwJSwgMSkgJ3N0cm9rZS13aWR0aD0nNScgLz4gIDwvZz48Zz4gPGxpbmUgeDE9JzMzOCcgeTE9JzM1My41JyB4Mj0nMzM5JyB5Mj0nMzcuNScgIHN0cm9rZS1saW5lY2FwPSdyb3VuZCcgc3Ryb2tlPSdoc2xhKDAsIDAlLCAwJSwgMSkgJ3N0cm9rZS13aWR0aD0nNScgLz4gIDwvZz48Zz4gPGxpbmUgeDE9JzMzOScgeTE9JzM3LjUnIHgyPSc2NCcgeTI9JzMyLjUnICBzdHJva2UtbGluZWNhcD0ncm91bmQnIHN0cm9rZT0naHNsYSgwLCAwJSwgMCUsIDEpICdzdHJva2Utd2lkdGg9JzUnIC8+ICA8L2c+PGc+IDxsaW5lIHgxPSczMDUnIHkxPSczOS41JyB4Mj0nMzA1JyB5Mj0nMzkuNScgIHN0cm9rZS1saW5lY2FwPSdyb3VuZCcgc3Ryb2tlPSdoc2xhKDAsIDAlLCAwJSwgMSkgJ3N0cm9rZS13aWR0aD0nNScgLz4gIDwvZz4gPC9nPiA8L3N2Zz4=
```

It appears that this reccuring pattern have an extra space inside a quoted expression while a space is missing between two XML attributes. For example :

```
stroke='hsla(0, 0%, 0%, 1) 'stroke-width='5'
```

I validated the fix by removing extra spaces and it is working:

```
<svg xmlns='http://www.w3.org/2000/svg' width='307' height='325' viewBox='0 0 307 325'> <rect width='307' height='325' x='0' y='0' fill='transparent' /> <g transform='translate(-32, -29)'> <g> <line x1='62' y1='33.5' x2='62' y2='277.5'  stroke-linecap='round' stroke='hsla(0, 0%, 0%, 1)' stroke-width='5' />  </g><g> <line x1='62' y1='276.5' x2='35' y2='316.5'  stroke-linecap='round' stroke='hsla(0, 0%, 0%, 1)' stroke-width='5' />  </g><g> <line x1='65' y1='329.5' x2='65' y2='354.5'  stroke-linecap='round' stroke='hsla(0, 0%, 0%, 1)' stroke-width='5' />  </g><g> <line x1='65' y1='354.5' x2='338' y2='353.5'  stroke-linecap='round' stroke='hsla(0, 0%, 0%, 1)' stroke-width='5' />  </g><g> <line x1='338' y1='353.5' x2='339' y2='37.5'  stroke-linecap='round' stroke='hsla(0, 0%, 0%, 1)' stroke-width='5' />  </g><g> <line x1='339' y1='37.5' x2='64' y2='32.5'  stroke-linecap='round' stroke='hsla(0, 0%, 0%, 1)' stroke-width='5' />  </g><g> <line x1='305' y1='39.5' x2='305' y2='39.5'  stroke-linecap='round' stroke='hsla(0, 0%, 0%, 1)' stroke-width='5' />  </g> </g> </svg>
```

**Comment by irskep** [comment]
Updated title. Looks like there's an error specifically with the line shape. On lines 98 and 99, `capString` should be inside the tag but isn't.

**Comment by irskep** [comment]
Fixed in 0.5.0
